src/status_im2/contexts/shell/activity_center/notification/reply/view.cljs

```diff
@@ -54,11 +61,11 @@
 (defn view
   [{:keys [notification set-swipeable-height customization-color] :as props}]
   (let [{:keys [author chat-name community-id chat-id
-                message read timestamp]} notification
-        community-chat?                  (not (string/blank? community-id))
-        community                        (rf/sub [:communities/community community-id])
-        community-name                   (:name community)
-        community-image                  (get-in community [:images :thumbnail :uri])]
+                message read timestamp albumMessages]} notification
```

Comment:
It's good that we rename the `albumMessages` key to `album-messages` (kebab case) when we rename the rest of the notification keys.
https://github.com/status-im/status-mobile/blob/a99ae74da7373328fc8378bbbc9eec2ac0a6e94e/src/status_im/data_store/activities.cljs#L55-L68